web-registry/src/pages/Dashboard/MyProjects/MyProjects.tsx

```diff
@@ -87,24 +61,53 @@ const MyProjects = (): JSX.Element => {
           <Grid item xs={12} md={6} lg={4}>
             <CreateProjectCard
               isFirstProject={isFirstProject}
-              onClick={submitCreateProject}
+              onClick={() =>
+                submitCreateProject({
+                  createProject,
+                  createWallet,
+                  setError,
+                  navigate,
+                  wallet,
+                  walletData,
+                })
+              }
+              sx={{ height: { xs: '100%' } }}
             />
           </Grid>
         )}
-        {/* TODO: ProjectCards used below temporarily. Will probably be a new variation for this purpose */}
         {isProjectAdmin &&
-          projects?.map((project, i) => (
-            <Grid key={i} item xs={12} md={6} lg={4}>
-              <ProjectCard
-                name={project?.handle || project?.id}
-                imgSrc={''}
-                place="TODO"
-                area={0}
-                areaUnit="ha"
-                track={track}
-              />
-            </Grid>
-          ))}
+          projects?.map((project, i) => {
+            const currentProject = projectsWithOrders.find(
+              projectsWithOrder => projectsWithOrder.id === project?.onChainId,
+            );
+            const name =
+              currentProject?.name ??
+              project?.metadata?.['schema:name'] ??
+              project?.handle ??
+              project?.id;
+            return (
+              <Grid key={i} item xs={12} md={6} lg={4}>
+                <WithLoader isLoading={isLoadingProjects} variant="skeleton">
+                  <ProjectCard
+                    {...DEFAULT_PROJECT}
+                    {...(currentProject ?? {})}
+                    name={name}
+                    onButtonClick={() => {
+                      if (project?.onChainId) {
+                        navigate(
+                          `/project-pages/${project?.onChainId}/edit/basic-info`,
+                        );
+                      } else {
+                        // TODO: update to navigate to the step the user left off the flow instead of /basic-info
```

Comment:
```suggestion
                        // TODO: update to navigate to the step the user left off the flow instead of /basic-info
                        // https://github.com/regen-network/regen-registry/issues/1574
```